app/process/model/transaction/txid.go

```diff
@@ -22,7 +22,6 @@ import (
 	"strings"
 )
 
-// TODO not necessary -> remove
 type TxId struct {
```

Comment:
This code is used only in the e2e tests. I think that we have 2 options:
- Remove the code from here and in the e2e tests add a util function for the `FromHederaTransactionID`
- Move this code to `encoding` and use it in the `e2e` tests and the `mirror-node` client (the function equivalent to this logic is `parseIntoQueryableTx`